DEV_SETUP_MAC.md

```diff
@@ -141,7 +141,7 @@ Note: `kafka` will be very cranky on start up. You might have to restart it if y
 
 ### Installing and running Elasticsearch 6.8.23 outside of Docker
 
-First, ensure that you have Java 8 running. `java -version` should output something like `openjdk version "1.8.0_322"`.
+First, ensure that you have Java 13 running. `java -version` should output something like `openjdk version "17.0.7" 2023-04-18 LTS"`.
```

Comment:
Why Java 13? I would have assumed Java 17 would be our recommendation since that is what we are using right?

Reply:
Sure a typo. Thanks for catching that , fixed in https://github.com/dimagi/commcare-hq/pull/36468/commits/7f08893e061206b4fdce96d6339bd22c3a6672b2